AppRoot.js

```diff
@@ -28,7 +28,13 @@ export default class AppRoot extends Component {
       }
 
       if (user) {
-        this.router.navigate('Conversations')
+        firebase.userNameExists().then(userNameExists => {
```

Comment:
Why don't you rewrite this with `await firebase.userNameExists()` as we do everywhere else?